FeathrSandbox.Dockerfile

```diff
@@ -55,8 +59,8 @@ COPY --chown=1000:100 ./docs/samples/local_quickstart_notebook.ipynb .
 COPY --chown=1000:100 ./feathr-sandbox/feathr_init_script.py .
 
 # Run the script so that maven cache can be added for better experience. Otherwise users might have to wait for some time for the maven cache to be ready.
-RUN python feathr_init_script.py
-RUN python -m pip install interpret
+# RUN python feathr_init_script.py
```

Comment:
Why are they commented out?

Reply:
To optimize build speed, temporarily commented out, will remove it later.

Reply:
TODO has already been added.